src/Whisper.jl

```diff
@@ -141,23 +141,18 @@ end
 
 function transcribe(
     file_path::String, srt_path::String;
-    model_name::String = "tiny.en", language::String = "english",
+    model_name::String = "tiny.en", language::Maybe{String} = nothing,
     dev = gpu, precision = f16,
 )
-    if language != "english" && endswith(model_name, ".en")
+    multilingual = !endswith(model_name, ".en")
+
+    if (language ≢ nothing && language != "english") && multilingual
```

Comment:
should it not be `&& !multilingual` ?

Reply:
Yes, I fixed it in the next commit:
https://github.com/JuliaNeuralGraphics/Whisper.jl/pull/3/commits/79c476d9a1c0ca9702747c4a5b3d52002fda79b6

Reply:
oops, sorry